include/windows/hax_windows.h

```diff
@@ -116,10 +116,10 @@ static inline void hax_mutex_free(hax_mutex lock)
 }
 
 /* Return true if the bit is set already */
-static int hax_test_and_set_bit(int bit, uint64_t *memory)
+static int hax_test_and_set_bit(uint64 bit, uint64_t *memory)
```

Comment:
I have concerns about this change. Is it safe to use a 64-bit integer as an array index (`base`) on a 32-bit OS? (We still support 32-bit Windows.) We could change `int` to `long`, but `long` is 32 bits on Windows. If the caller could adapt to the current interface, that would be great. After all, the protection bitmap shouldn't be very long, right? If max_gpa is 8GB, the bitmap size will be 256KB.
Anyway, since this change is self-contained, if it's really necessary, we should make a separate patch for it.

Reply:
Removed.